README.md

```diff
@@ -1,2 +1,158 @@
-# docker-php
+# PHP-FPM
+
+[![Docker Stars](https://img.shields.io/docker/stars/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![Docker Pulls](https://img.shields.io/docker/pulls/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![microbadger version](https://images.microbadger.com/badges/version/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own version badge on microbadger.com")
+[![microbadger image](https://images.microbadger.com/badges/image/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com")
+
+# Description
+
 Extends official PHP Docker images with extensions and tools to be able to run Spryker on.
+
+* Based on: `Alpine 3.9`
+* Users: `root`, `spryker`
+* Working directory: `/data`
+* Includes:
+  * Composer
+  * `hirak/prestissimo`
+  * `aws/aws-sdk-php`
+
+> Note: Provided images require additional configuration for development, staging and production use.
+
+## Tags
+
+| Tag     | PHP version     | Details     | Dockerfile     |
+| :------------- | :------------- | :------------- | :------------- |
+| [spryker/php:latest](https://hub.docker.com/r/spryker/php/tags) | 7.2.16 | [![](https://images.microbadger.com/badges/image/spryker/php:latest.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.2/Dockerfile) |
+| [spryker/php:7.3](https://hub.docker.com/r/spryker/php/tags)  | 7.3.3 | [![](https://images.microbadger.com/badges/image/spryker/php:7.3.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.3/Dockerfile) |
+| [spryker/php:7.2](https://hub.docker.com/r/spryker/php/tags)  | 7.2.16 | [![](https://images.microbadger.com/badges/image/spryker/php:7.2.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.2/Dockerfile) |
+| [spryker/php:7.1](https://hub.docker.com/r/spryker/php/tags)  | 7.1.27 | [![](https://images.microbadger.com/badges/image/spryker/php:7.1.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.1/Dockerfile) |
+
+
+## How to use
+
+### Pull image
+```bash
+$ docker pull spryker/php
```

Comment:
What is the reason to provide these two lines in sample? Would be enough to have only one of them?

Reply:
I want to pull both images